metrics/storage/webtooling.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+#
+# Copyright (c) 2020-2021 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'web tooling benchmark'
+# https://github.com/v8/web-tooling-benchmark
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+TEST_NAME="web-tooling"
+IMAGE="docker.io/library/local-web-tooling:latest"
+DOCKERFILE="${SCRIPT_PATH}/web-tooling-dockerfile/Dockerfile"
+CI_JOB="${CI_JOB:-""}"
+configuration_file="/usr/share/defaults/kata-containers/configuration.toml"
```

Comment:
not being used

Reply:
@chavafg is being used for the generation of the json

Reply:
in one of the metrics_json .....